### PR TITLE
[core][computes] Improved error handling in sync code

### PR DESCRIPTION
**Pull request by amitsrivastava**
Added a try/except to log error and continue if one of the namespaces runs into and error.

Change-Id: I25d5ba71e0d6032e513465de1fa616dfe4ea0878

### CODE REVIEW

**Review by athithyaaselvam — APPROVED** [review]
LGTM!

File: desktop/libs/notebook/src/notebook/connectors/base.py [hunk 1]
```diff
@@ -437,7 +437,7 @@ def get_api(request, snippet):
     interpreter = snippet.get('interpreter')
   elif get_cluster_config(request.user).get('has_computes'):
     if snippet.get('type') in ('hive-compute', 'impala-compute') and snippet.get('id'):
-      LOG.debug("Loading the compute from db using snippet['id']: %" % snippet['id'])
+      LOG.debug("Loading the compute from db using snippet['id']: %s" % snippet['id'])
```
Comment (Harshg999) [inline desktop/libs/notebook/src/notebook/connectors/base.py:440]: Nice catch!

File: desktop/core/src/desktop/management/commands/sync_warehouses.py [hunk 2]
```diff
@@ -95,22 +95,26 @@ def get_computes_from_k8s():
...
-      computes.append(item)
-      populate_impala(namespace, item)
+    try:
+      namespace = n.metadata.name
+      LOG.info('Getting details for ns: %s' % namespace)
+      item = {
+        'name': n.metadata.labels.get('displayname'),
+        'description': '%s (%s)' % (n.metadata.labels.get('displayname'), n.metadata.name),
+        'external_id': namespace,
+        #'creation_timestamp': n.metadata.labels.get('creation_timestamp'),
```
Comment (Harshg999) [inline desktop/core/src/desktop/management/commands/sync_warehouses.py:105]: Nit: to remove or its a to-do?

**Review by Harshg999 — APPROVED** [review]
LGTM!

**Pull request merged**